File: Library_API/BusinessLogicLayer/DTOs/LibraryRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BusinessLogicLayer.DTOs
{
    public record LibraryRequest(
        string id,
        string description,
        string coverImageUrl,
        string name,
        string address,
        string phone,
        List<string> books);

}

File: Library_API/UI/library-app/src/app/interfaces/library-request.ts
import {ScheduleRequest} from './schedule-request';

export interface LibraryRequest {
    name: string,
    description: string,
    coverImageUrl: string,
    address: string,
    phone: string,
    email: string,
    studyRooms: number,
    computers: number,
    schedule: ScheduleRequest[]
}
